### ISSUE TITLE
🐛 [bug]: lint does not recognise new github concurency queue: max

### ISSUE THREAD

**Issue by jprecuch** [raven-actions/actionlint#64]
### 🐛 What happened?

https://docs.github.com/en/actions/how-tos/write-workflows/choose-when-workflows-run/control-workflow-concurrency
```
Check failure on line R686
Check failure:
unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"
```

### 🔬 How to reproduce?

Try to add the new concurency 
```
concurrency:
  group: auto-update-promotion-branch-${{ github.ref }}
  cancel-in-progress: false
  queue: max
```

### 🏗️ Code Sample / Log

```Shell

```

### 🌌 Environment (actionlint action)

latest

### 🌌 Environment (actionlint)

latest

### 🌌 Environment (GitHub Runner OS)

Linux

### 📜 Code of Conduct

- [x] I agree to follow this project's Code of Conduct.

**Comment by DariuszPorowski** [comment]
@jprecuch please report this issue to the `actionlint` itself: https://github.com/rhysd/actionlint
This project it's just a wrapper on top of cli, and the reported issue its not in the scope of this project.

**Comment by jprecuch** [comment]
It seems it was reported already https://github.com/rhysd/actionlint/issues/657
